README.md

```diff
@@ -1 +1,54 @@
-# config-monorepo
\ No newline at end of file
+# Eslint
+
+Provide reusable typescript and react eslint configs
+
+1. Install the peerDependencies of eslint configs:
+
+```sh
+npx install-peerdeps --dev @arbitrum/eslint-config-typescript/typescript
+npx install-peerdeps --dev @arbitrum/eslint-config-typescript/react
+```
+
+2. Usage
+
+```ts
+module.exports = {
+  extends: [
+    "@arbitrum/eslint-config-typescript/typescript",
+    "@arbitrum/eslint-config-typescript/react",
```

Comment:
A couple of nitpickings/questions: Should we maybe rename the package to `@arbitrum/typescript-eslint-config`, to better match `@typescript-eslint` and is it possible to do:

```suggestion
    "@arbitrum/typescript-eslint-config",
    "@arbitrum/typescript-eslint-config/react",
```

i.e. not having to do `/typescript` for the base package?

Reply:
If not, maybe just call it `/base`? 🤷

Reply:
having `eslint-config` at the start is standard, so I would keep it that way. `/base` make sense yes